src/main/java/io/vertx/core/spi/transport/Transport.java

```diff
@@ -174,6 +174,15 @@ default void configure(NetServerOptions options, boolean domainSocket, ServerBoo
     bootstrap.option(ChannelOption.SO_REUSEADDR, options.isReuseAddress());
     if (!domainSocket) {
       bootstrap.childOption(ChannelOption.SO_KEEPALIVE, options.isTcpKeepAlive());
+      if (options.isTcpKeepAlive() && options.getTcpKeepAliveIdleSeconds() != -1) {
+        bootstrap.childOption(EpollChannelOption.TCP_KEEPIDLE, options.getTcpKeepAliveIdleSeconds());
```

Comment:
Epoll options must be configured in EpollTransport subclass and not in this one, further more that forces users that do not use epoll to have the native epoll jar on their classpath

Reply:
Done